### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by vkatsuba**
Updated  for `.gitignore` few files which can be created and used in development.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@
 # diameter compiler atrifacts
 /src/diameter_*
 /include/diameter_*
+rebar3
```
Comment (ebengt) [inline .gitignore:12]: I thought we talked about this and said that nobody would put rebar3 in the repo?

Comment (vkatsuba) [reply .gitignore:12]: I use `rebar3` binary in repo during development, and for me it is a little inconvenience to care that it is in the root folder or not during commits

Comment (RoadRunnr) [reply .gitignore:12]: on Linux, putting in `~/bin` might be more convenient (create the directory if needed)

**Review by RoadRunnr — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (vkatsuba) [reply .gitignore:12]: Yes, thanks, I know it, but I use different `rebar` versions and by this reason I not put it into `~/bin`

**Pull request merged**